geoportailv3/static/js/draw/drawdirective.js

```diff
@@ -234,8 +257,8 @@ app.DrawController = function($scope, ngeoDecorateInteraction, ngeoLocation,
       function(evt) {
         goog.asserts.assertInstanceof(evt.element, ol.Feature);
         var feature = evt.element;
-        goog.events.listen(feature, goog.events.EventType.CHANGE,
-            this.onFeatureChange_, undefined, this);
+        goog.events.listen(feature, app.DrawEventType.PROPERTYMODIFYEND,
+            this.onFeaturePropertyChange_, undefined, this);
```

Comment:
Could you please explain why relying on feature "change" events did not work for you?